lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -172,4 +183,12 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
   end
 end
 
+function M.make_directory_change_for_command(dir, command)
+  if fn.has('win32') == 1 then
+    return string.format("pushd %s & %s & popd", dir, command)
```

Comment:
We did not solve that issue yet, whether to use pushd or cd on Windows, didn't we?

Reply:
`cd %s;\n` does not work on Windows :) It does not like that `;`

Reply:
I mean now we can test it quite easily in CI :)

Reply:
But bash on windows should behave like bash on Linux?

Reply:
you mean running this installation command from `bash` in CI?

Reply:
I mean when users on windows use bash instead of powershell. Will they have a problem? I can't really tell what's the common eniviroment for running nvim on windows (probably neovim-qt) but users seem to have managed to install parsers also with cd;

Reply:
I mean probably most of the shell commands which do check `win32` will not work on bash.